### PR TITLE
Separate bug, feature request, and improvement request issue templates

### PR DESCRIPTION
**Pull request by zrhoffman**
## What does this PR (Pull Request) do?

This PR makes it so you are prompted for the issue type (bug, feature, or improvement) when creating an issue instead of needing to delete irrelevant parts from the issue template depending on which type of issue you are creating.

- [x] This PR is not related to fixing any Issue 

## Which Traffic Control components are affected by this PR?

- Issue templates

## What is the best way to verify this PR?

1. Push the branch to your fork
```shell
git fetch zrhoffman
git checkout -tb PR-5047 zrhoffman/master
git push <your fork> PR-5047:master
```

2. Go to https://github.com/your-username/trafficcontrol/issues and click *New issues*

3. Form opinions

## The following criteria are ALL met by this PR

- [x] Markdown files changed only, no tests necessary
- [x] This PR includes documentation
- [x] An update to CHANGELOG.md is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-bug-report.md [hunk 1]
```diff
@@ -29,22 +30,24 @@ guidelines at https://www.apache.org/security/ regarding vulnerability disclosur
...
-If the current behavior is a bug or you can illustrate your feature request better with an example,
-please provide the *STEPS TO REPRODUCE* and include the applicable TC version.
+If the current behavior is a bug, please provide the *STEPS TO REPRODUCE* and
+include the applicable TC version.
 -->
 
 ## Anything else:
-<!-- e.g. stacktraces, related issues, suggestions how to fix -->
+<!-- e.g. stacktraces, related issues, suggestions how to fix (feel free to
+delete this section) -->
```
Comment (rawlinp) [inline .github/ISSUE_TEMPLATE/1-bug-report.md:50]: nit: with multi-line comments, I find it easier to parse when the `<!--` and `-->` are on separate lines like this:
```
<!--
This is an
example multi-line
comment.
-->
```

Sometimes I find myself accidentally changing text inside a comment (usually the "what does this PR do" section).

Comment (zrhoffman) [reply .github/ISSUE_TEMPLATE/1-bug-report.md]: Put `<!--` and `-->` on different lines for multiline comments in 1072cb9d5f.

**Review by rawlinp — APPROVED** [review]
I think this is great, but since this was brought up on the mailing list I'll wait a few days to merge so that others add feedback if they want.

**Comment by mitchell852** [comment]
image

Wondering if a "tech debt" button would be a good idea.

**Comment by zrhoffman** [comment]
Definition expanded in cfa96573c

**Review by mitchell852 — APPROVED** [review]
LGTM. nice work!

**Pull request merged**